src/lib/statistics/table_statistics.hpp

```diff
@@ -52,12 +53,20 @@ class TableStatistics final {
   TableStatistics estimate_disjunction(const TableStatistics& right_table_statistics) const;
   /** @} */
 
+  // Increases the (approximate) count of invalid rows in the table (caused by deletes).
+  void increment_invalid_row_count(uint64_t count);
```

Comment:
Isn't "increment" in steps of one?

Reply:
This is how it was called before. Would "increase" be more appropriate in your opinion?

Reply:
yes